test/plots/piePlotTests.ts

```diff
@@ -244,6 +244,27 @@ describe("Plots", () => {
       Plottable.Utils.Window.warn = oldWarn;
       svg.remove();
     });
+
+    describe("Labels", () => {
+      it("labels are shown and hidden appropriately", () => {
+        piePlot.removeDataset(simpleDataset);
+        var data = [
+          { key: "A", value: 1 }, { key: "B", value: 50 },
+          { key: "C", value: 1 }, { key: "D", value: 50 },
+          { key: "E", value: 1 }, { key: "F", value: 50 }
+        ];
+        var dataset = new Plottable.Dataset(data);
+        piePlot.addDataset(dataset).labelsEnabled(true);
+        $(".label-area").children("g").each(function(i) {
```

Comment:
Only need first level children for this, the children wrap <g> elements and it can get wonky.